event.py

```diff
@@ -1,20 +1,25 @@
+"""
+Implementing CalicoBot events
+"""
 import discord
 import datetime
 from discord.ext import commands
-from discord import app_commands
 import time
-import asyncio
-import difflib
-import main
 
-bot = main.bot
+bot = None
+def event_init(main_bot):
+    bot = main_bot
```

Comment:
> **Note**
> You should apply this suggestion after changing the variable name.

I wasn't understanding that It can't assign outside variables.
```suggestion
    global BOT# pylint: disable=global-statement
    BOT = main_bot
```